Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/Climber.java

```diff
@@ -11,18 +12,28 @@
  */
 public class Climber extends Subsystem {
 
-	 public WPI_TalonSRX climbMotor;
+	private WPI_TalonSRX climbMotor;
+	public Object climbDown;
 	
 	public Climber() {
 	super();
 	
-	climbMotor = new WPI_TalonSRX(RobotMap.climber);
+	climbMotor = new WPI_TalonSRX(RobotMap.CLIMBER);
     // Put methods for controlling this subsystem
     // here. Call these from Commands.
 	 }
     public void initDefaultCommand() {
         // Set the default command for a subsystem here.
         //setDefaultCommand(new MySpecialCommand());
     }
+    public void climbUp() {
```

Comment:
`Robot.climber.climbMotor` is not needed in these methods. Because we are already in the `Climber` subsystem, we have access to the `climbMotor`. So you can drop `Robot.climber.` and just use `climbMotor`